dbms/src/Storages/Page/V3/Universal/UniversalPageIdFormatImpl.h

```diff
@@ -164,6 +165,13 @@ struct UniversalPageIdFormat
         return buff.releaseStr();
     }
 
+    // LOCAL_PREFIX type keyspace_id(64 bits)
+    static String toEncryptionKeyPageID(KeyspaceID keyspace_id)
+    {
+        // use keyspace_id as region_id
+        return toLocalKVPageID(LocalKVKeyType::EncryptionKey, keyspace_id);
```

Comment:
Don't "use keyspace_id as region_id", but use seperated functions?
toLocalPageId could be change to `LOCAL_PREFIX type`, with name `toLocalPagePrefix`